src/site/content/en/accessible/control-focus-with-tabindex/index.md

```diff
@@ -142,6 +142,113 @@ You can learn more about them in our guide on
+because they're the only native HTML form elements.
+To insert other elements into the tab order, you would add a `tabindex` attribute.
+{% endAssessmentHint %}
+
+{% endTab %}
+{% Tab 'sample' %}
+
+```html
+<section tabindex="-1">
+  <h2>Cat facts</h2>
+  <ul>
+    <li>A group of cats is called a <a href="https://m-w.com/dictionary/clowder">clowder</a>.</li>
+    <li>Most cats are <a href="https://www.catfacts.org/catnip.html"> unaffected by catnip</a>.</li>
+  </ul>
+</section>
+```
+
+{% AssessmentHint 'Which elements from the sample are included in the tab order?' %}
+Only the `<a>` elements are included in the tab order.
+
+The `<section>` element is not in the tab order
+because it has a negative `tabindex` value.
+(It can, however, be focused using the `focus()` method.)
+{% endAssessmentHint %}
+
+{% endTab %}
+{% Tab 'sample' %}
+
+This HTML renders a popup menu followed by a search input:
+
+```html
+<div role="menu" tabindex="0">
+  <a role="menuitem" href="/learn/" tabindex="-1">Learn</a>
+  <a role="menuitem" href="/measure/" tabindex="-1">Measure</a>
+  <a role="menuitem" href="/blog/" tabindex="-1">Blog</a>
+  <a role="menuitem" href="/about/" tabindex="-1">About</a>
+</div>
+<input tabindex="1" type="text" role="search" aria-label="Search" placeholder="Search">
+```
+
+{% AssessmentHint 'Which element in the sample comes first in the tab order?' %}
+The **Search** text input comes first in the tab order.
+Because it has a `tabindex` greater than 1, it jumps to the front of the tab order.
+
+(This behavior is likely to cause confusion
+if the menu is positioned on the page before the search input.
+This is an example of why having a `tabindex` value greater than zero
+is considered an anti-pattern.)
+
+{% endAssessmentHint %}
+
+{% endTab %}
+{% Tab 'sample' %}
+
+This HTML renders a custom radio group:
+
+```html
+<div role="radiogroup" aria-labelledby="breed-header">
+  <h3 id="breed-header">Your cat's breed</h3>
+  <div role="radio" aria-checked="false" tabindex="0">Persian</div>
+  <div role="radio" aria-checked="false" tabindex="-1">Bengal</div>
+  <div role="radio" aria-checked="false" tabindex="-1">Maine Coon</div>
+</div>
+```
+
+To keep things simpler, ignore the
+[`aria-*` and `role` attributes](/semantics-and-screen-readers) for now.
+
+{% AssessmentHint 'For a roving `tabindex`, what logic should run when a user presses the `Right` arrow key while a radio element is focused?' %}
+- Change the `tabindex` values for all radio elements in the group to -1.
+- If there's a radio element after the one that's focused,
+  set its `tabindex` value to 0.
+- If there's no radio element after the one that's focused,
+  set the `tabindex` value of the first radio element in the group to 0.
+- Focus the radio element that now has a `tabindex` of 0.
+
+That's a lot—and it doesn't even include ARIA attributes!
+This is an example of why it's easier to use native elements
+with built-in keyboard behavior whenever you can.
```

Comment:
Dang, pretty effective way to demonstrate why custom tabindex logic is a hassle. Because as I was reading through the list I was like "this is complicated"